### DISCUSSION TITLE
How to block by name of "known filter" or "category" or "source" in user rules?

### DISCUSSION THREAD

**Discussion by user9931** [AdguardTeam/AdGuardHome#6125] [Q&A]
Creating in "Q&A" because maybe i just cant find this setting...

For some reason cant upload an image (?), but anyway...

In the logs there's some info like that

Request details
Time
13:13:31.449
Date
21.08.2023
Domain
conn-service-cn-05.allawntech.com
Type
A
Protocol
Plain DNS
Known tracker
Name
OPPO
Category
Mobile Analytics
Source
[AdGuard](https://github.com/AdguardTeam/AdGuardHome/tree/master/client/src/helpers/trackers/trackers.json)

It is possible to block|unblock|rewrite any requrests that match criteria like category "Mobile Analytist" or/and from this source ("Adguard") or/and Known tracker with name "OPPO"? read https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists and dont see how to do it...  If it really dont possible then maybe move it into issues...

**Comment by ainar-g** [comment]
Not currently, as this information exists only in the UI.  You can create a feature request, and we'll see if it gets popular.
